app/src/main/res/xml/preferences.xml

```diff
@@ -80,6 +80,11 @@
             android:icon="@drawable/ic_phone_check"
             android:title="@string/keep_screen_on"
             android:summary="@string/keep_screen_on_def"/>
+        <SwitchPreference
+            android:key="pinch_to_zoom"
+            android:icon="@drawable/ic_phone_check"
```

Comment:
Minor comment but maybe a more appropriate icon so it can look different from the option directly above it?